src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements CalculateArea {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract void draw();

```

Comment:
Don’t use abstract classes to set behaviour for classes
Abstract classes and interfaces have different use cases. Try to figure out when to use both in this task by yourself. If you’re blocked [this](https://stackoverflow.com/a/479168) may give you a hint.